src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../../styles/styles';
+import SidebarLinks from './SidebarLinks';
+import CreateMenu from '../../../components/CreateMenu';
+import FAB from '../../../components/FAB';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+
+const propTypes = {
+    // Toggles the navigation menu open and closed
+    onLinkClick: PropTypes.func.isRequired,
+
+    // Current state of the CreateMenu component (active or inactive)
+    isCreateMenuActive: PropTypes.bool.isRequired,
+
+    // Callback to fire on request to toggle the CreateMenu
+    toggleCreateMenu: PropTypes.func.isRequired,
+
+    // Callback to fire when a CreateMenu item is selected
+    onCreateMenuItemSelected: PropTypes.func.isRequired,
+
+    // Callback to fire on avatar click
+    onAvatarClick: PropTypes.func.isRequired,
+};
+
+const SidebarScreen = props => (
+    <ScreenWrapper
+        includePaddingBottom={false}
+    >
+        {insets => (
```

Comment:
Since it is an arrow function with an implicit return adding parens here will break style. If we had curly braces and an explicit `return` then we would use the parens.

Reply:
Ahh right right, that makes sense